controllers/operandrequest/reconcile_operand.go

```diff
@@ -179,6 +179,63 @@ func (r *Reconciler) reconcileOperand(ctx context.Context, requestInstance *oper
 
 // reconcileCRwithConfig merge and create custom resource base on OperandConfig and CSV alm-examples
 func (r *Reconciler) reconcileCRwithConfig(ctx context.Context, service *operatorv1alpha1.ConfigService, namespace string, csv *olmv1alpha1.ClusterServiceVersion) error {
+	merr := &util.MultiErr{}
+
+	// Create k8s resources required by service
+	if service.Resources != nil {
+		for _, res := range service.Resources {
+			if res.APIVersion == "" {
+				return fmt.Errorf("The APIVersion of k8s resource is empty for operator " + service.Name)
+			}
+
+			if res.Kind == "" {
+				return fmt.Errorf("The Kind of k8s resource is empty for operator " + service.Name)
+			}
+			if res.Name == "" {
+				return fmt.Errorf("The Name of k8s resource is empty for operator " + service.Name)
+			}
+			var k8sResNs string
+			if res.Namespace == "" {
+				k8sResNs = namespace
+			} else {
+				k8sResNs = res.Namespace
+			}
+
+			var k8sRes unstructured.Unstructured
+			k8sRes.SetAPIVersion(res.APIVersion)
+			k8sRes.SetKind(res.Kind)
+			k8sRes.SetName(res.Name)
+			k8sRes.SetNamespace(k8sResNs)
+
+			err := r.Client.Get(ctx, types.NamespacedName{
+				Name:      res.Name,
+				Namespace: k8sResNs,
+			}, &k8sRes)
+
+			if err != nil && !apierrors.IsNotFound(err) {
+				merr.Add(errors.Wrapf(err, "failed to get k8s resource %s/%s", k8sResNs, res.Name))
+			} else if apierrors.IsNotFound(err) {
+				if err := r.createK8sResource(ctx, k8sRes, res.Data.Raw, res.Labels, res.Annotations); err != nil {
+					merr.Add(err)
+				}
+			} else {
+				if checkLabel(k8sRes, map[string]string{constant.OpreqLabel: "true"}) {
+					// Update k8s resource
+					klog.V(3).Info("Found existing k8s resource: " + res.Name)
+					if err := r.updateK8sResource(ctx, k8sRes, res.Data.Raw, res.Labels, res.Annotations, res.Force); err != nil {
```

Comment:
There is a point we need to discuss.

We expect `res.Force` by default to be true, but the default value of boolean is false, which will cause a misunderstanding if users don't set it or set it to false.

I am thinking if we can use the `force` in `*bool` type.

Reply:
Also, could we add the check for `res.Force` at the same level as `checkLabel`? Then we don't need to include it in the update function

Reply:
Yeah, agree. I previously think the default value for `Force` is `false`. I will update the `*bool` type for `Force`.

> Could we add the check for res.Force at the same level as checkLabel? Then we don't need to include it in the update function

I am not sure it is reasonable...... If the `Force` is false, but the config of k8s resource is updated by users, in this case, we should still update the resources, right?

The path that we will not update the resource is either `checkLabel == false` or ` isEqual && Force == false`

Reply:
It seems that we could not use `*bool` as the type, because we [could not get the address of a constant.](https://stackoverflow.com/questions/42594789/initialize-string-pointer-in-struct)